explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -79,14 +80,25 @@ case class TargetTabContents(
   expandedIds:      View[SortedSet[ObsIdSet]],
   readonly:         Boolean
 ) extends ReactFnProps(TargetTabContents.component):
-  val targets: UndoSetter[TargetList] = programSummaries.zoom(ProgramSummaries.targets)
-
-  val obsAndTargets: UndoSetter[ObservationsAndTargets] =
+  private val targets: UndoSetter[TargetList] = programSummaries.zoom(ProgramSummaries.targets)
+
+  private def sitesForTarget(targetId: Target.Id): List[Site] =
+    programSummaries.get.targetObservations
+      .get(targetId)
+      .foldMap: obsIds =>
+        obsIds.toList
+          .map: obsId =>
+            programSummaries.get.observations
+              .getValue(obsId)
+              .flatMap(_.observingMode.map(_.siteFor))
```

Comment:
so this is the preferred site but there are targets visible from  either side. I guess it is nit-picking